src/content/docs/new-relic-solutions/new-relic-one/introduction-new-relic-platform.mdx

```diff
@@ -0,0 +1,216 @@
+
+import transtionGuideUiRedesignCustomize from 'images/transtion-guide_screenshot-full_ui-redesign-customize.gif'
+
+import entityExplorer from 'images/accounts_screenshot-crop_entity-explorer.png'
+
+import navigator from 'images/accounts_screenshot-crop_navigator.png'
+
+import lookout from 'images/accounts_screenshot-crop_lookout.png'
+
+import search from 'images/accounts_screenshot-crop_search.png'
+
+import chartOptions from 'images/apm_screenshot-crop_chart-options.png'
+
+import traceWaterfall from 'images/apm_screenshot-crop_trace-waterfall.png'
+
+import serviceMap from 'images/apm_screenshot-full_service-map.png'
+
+import workloadMap from 'images/workloads_screenshot-full_workload-map.png'
+
+import queriesnrqlNrqlExampleTimeseries from 'images/queries-nrql_screenshot-crop_nrql-example-timeseries.png'
+
+import accountsUserMenuLocation from 'images/accounts_screenshot-full_user-menu-location.png'
+
+
+Now that you've [created a New Relic account](https://newrelic.com/signup), and [installed some observability tools](/docs/new-relic-solutions/get-started/implementation-guide-intro), you probably want some tips for getting around the platform. 
+
+To access New Relic, go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities)**. If you have an EU datacenter account, go to **[one.eu.newrelic.com](https://one.eu.newrelic.com)**.
+
+Here we've got a 15 minute video that will show you some of the more important New Relic experiences. Note that this video is from the point of view of a [full platform user](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type). 
+
+<Video
+  type="wistia"
+  id="y7cea3r0a0"
+/>
+
+Next we'll give you some details and links on the topics covered in this video.
+
+## Capabilities [#capabilities]
+
+If you're new to using New Relic, your left sidebar will likely be empty. To remedy that, you can go to the [**All capabilities** page](https://one.newrelic.com/all-capabilities). That's where you'll find our core features and experiences. You can click the pin icon to pin them to your left-side navigation. 
+
+<img
+  title="Customize New Relic"
+  alt="A gif demonstrating the New Relic redesign's customization features."
+  src={transtionGuideUiRedesignCustomize}
+/>
+
+Here are some recommended capabilities to pin: 
+
+* **APM and services**: Most New Relic customers use APM, so you'll probaby be pinning that. 
+* **Infrastructure**: Most customers use infrastructure monitoring, too. 
+* **Distributed tracing**: Most customers use distributed tracing to see connections between their monitored services and hosts.  
+* **Logs**: Most customers use our log management UI to dig down into the source of a problem. 
+* **Query your data**: This is what you use to run queries of your data and create custom charts. 
+* **Dashboards**: Most customers create custom dashboards so it helps to have this pinned. 
+* **Workloads**: Workloads let you group together related monitored entities. 
```

Comment:
This section feels a little disjointed to me. Some of them seem super casual (most people do this), some of them seem like they're very descriptive (workloads), others feel kind of in the middle. Maybe redo them to be like workloads?